akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -98,6 +98,16 @@ object Sink {
   def headOption[T]: Sink[T, Future[Option[T]]] = new Sink(new HeadOptionSink[T](DefaultAttributes.headSink, shape("HeadOptionSink")))
 
   /**
+   * TODO: document usage
+   *
+   */
+  def toSeq[T]: Sink[T, Future[Seq[T]]] = {
```

Comment:
I think that would clash with GlowOps.collect. Perhaps collection?

-- 
Cheers,
√
On 27 Nov 2015 18:09, "Roland Kuhn" <notifications@github.com> wrote:

> In akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala
> <https://github.com/akka/akka/pull/18965#discussion_r46062178>:
>
> > @@ -98,6 +98,16 @@ object Sink {
> >    def headOption[T]: Sink[T, Future[Option[T]]] = new Sink(new HeadOptionSink[T](DefaultAttributes.headSink, shape("HeadOptionSink")))
> >
> >    /**
> > +   * TODO: document usage
> > +   *
> > +   */
> > +  def toSeq[T]: Sink[T, Future[Seq[T]]] = {
>
> how about calling it collect? it does return a “collect-ion” as well …
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/18965/files#r46062178>.
>